crates/emitter/src/emitter.rs

```diff
@@ -124,6 +124,17 @@ impl From<BytecodeOffset> for usize {
     }
 }
 
+pub struct BytecodeOffsetDiff {
+    pub diff: i32,
```

Comment:
it would be nice to implement `From<BytecodeOffsetDiff>` for `i32`, and make this field private.